Tools/autotest/common.py

```diff
@@ -1528,6 +1531,29 @@ def wait_ekf_happy(self, timeout=30, require_absolute=True):
         raise AutoTestTimeoutException("Failed to get EKF.flags=%u" %
                                        required_value)
 
+    def wait_gps_disable(self, timeout=30):
+        """Wait for EKF to be worried of GPS"""
+        self.set_parameter("SIM_GPS_DISABLE", 1)
+        tstart = self.get_sim_time()
+        # all of these must NOT be set for arming NOT to happen:
+        not_required_value = (mavutil.mavlink.ESTIMATOR_VELOCITY_HORIZ |
+                              mavutil.mavlink.ESTIMATOR_POS_HORIZ_REL |
+                              mavutil.mavlink.ESTIMATOR_POS_HORIZ_ABS |
+                              mavutil.mavlink.ESTIMATOR_PRED_POS_HORIZ_ABS)
```

Comment:
All of these? It should just need not having POS_HORIZ_REL. Why do you need all the others?

Reply:
I have just take the opposition of wait GPS

Reply:
Exactly my point - for arming, you need all those to be set; for not being able to arm, you just need one of them to not be set.

In my opinion it is more intuitive to just check POS_HORIZ_REL, but if you want to use either of them, you just need to change the check to use `current & not_required_value != not_required_value`.

Reply:
done